components/skin/ui/src/main/resources/PhenoTips/Skin.xml

```diff
@@ -794,10 +795,14 @@ h1:target, h2:target, h3:target {
   float: right;
 }
 .language-block .rightmenu a {
```

Comment:
Probably not entirely relevant for this PR, but IMO "Language" should be in a `<label>` for the `<select>`, not an `<a>` element.